sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -83,50 +83,71 @@ internal class NextLinkOperationImplementation : IOperation
 
         public static IOperation Create(
             HttpPipeline pipeline,
-            RehydrationToken? rehydrationToken,
-            string? apiVersionOverride = null)
+            RehydrationToken rehydrationToken,
+            bool skipApiVersionOverride = false,
```

Comment:
Are these version parameters ever passed to the method?  I couldn't find a place where they are.  If they are not used, it would be better to remove them and the logic that uses them in this method -- the logic looks incorrect to me, and if it's never used, it would be better to delete it now so it doesn't cause bugs later.

Reply:
removed the API versions